core-bundle/src/Resources/contao/classes/Frontend.php

```diff
@@ -582,9 +582,13 @@ protected function prepareMetaDescription($strText)
 	 * Return the cron timeout in seconds
 	 *
 	 * @return integer
+	 *
+	 * @deprecated Deprecated since Contao 4.9, to be removed in Contao 5.0.
 	 */
 	public static function getCronTimeout()
 	{
+		trigger_deprecation('contao/core-bundle', '4.9', 'Calling "%s::%s()" has been deprecated and will no longer work in Contao 5.0.', self::class, __METHOD__);
```

Comment:
```suggestion
		trigger_deprecation('contao/core-bundle', '4.9', 'Calling "%s()" has been deprecated and will no longer work in Contao 5.0.', __METHOD__);
```
`__METHOD__` already contains the class name.